alvr/audio/src/android.rs

```diff
@@ -0,0 +1,74 @@
+use alvr_common::anyhow::{bail, Context, Result};
+use alvr_session::{CustomAudioDeviceConfig, MicrophoneDevicesConfig};
+use cpal::traits::HostTrait;
+use rodio::DeviceTrait;
+
+use crate::{device_from_custom_config, AudioDevice};
+
+impl AudioDevice {
+    pub fn new_output(config: Option<&CustomAudioDeviceConfig>) -> Result<Self> {
+        let host = cpal::default_host();
+
+        let device = match config {
+            None => host
+                .default_output_device()
+                .context("No output audio device found")?,
+            Some(config) => device_from_custom_config(&host, config)?,
+        };
+
+        Ok(Self {
+            inner: device,
+            is_output: true,
+        })
+    }
+
+    pub fn new_input(config: Option<CustomAudioDeviceConfig>) -> Result<Self> {
+        let host = cpal::default_host();
+
+        let device = match config {
+            None => host
+                .default_input_device()
+                .context("No input audio device found")?,
+            Some(config) => device_from_custom_config(&host, &config)?,
+        };
+
+        Ok(Self {
+            inner: device,
+            is_output: false,
+        })
+    }
+
+    // returns (sink, source)
+    pub fn new_virtual_microphone_pair(config: MicrophoneDevicesConfig) -> Result<(Self, Self)> {
```

Comment:
I'm not sure what to do here, since this isn't needed on the client so it could just always return an error, but idk if it wouldn't be used in a potential future macos port

Reply:
Considering we need to stub the macos impl anyway, just remove this, we have git history